dask_cudf/tests/test_core.py

```diff
@@ -327,6 +327,14 @@ def gddf(gdf):
 def test_unary_ops(func, gdf, gddf):
     p = func(gdf)
     g = func(gddf)
+
+    # Fixed in https://github.com/dask/dask/pull/4657
+    if isinstance(p, cudf.Index):
+        from packaging import version
+        if version.parse(dask.__version__) < version.parse("1.1.6"):
```

Comment:
@mrocklin can you confirm this is the correct version with the fix?

Reply:
Sure.  That should work.